vendor/api/v1/authorizenet/views.py

```diff
@@ -162,38 +165,46 @@ class AuthorizeCaptureAPI(AuthorizeNetBaseAPI):
     API endpoint to get event notifications from authorizenet when a authcaputre is created.
     If there is a subscription tied to the transaction, it will renew such subscription
     """
+    def is_request_valid(self, site, request_data):
+        if not request_data:
+            logger.warning("AuthorizeCaptureAPI post: Webhook event has no body")
+            return False
+
+        if not request_data.get('payload').get('id'):
```

Comment:
might need to do `.get('payload', {}).get('id')` to make sure this doesn't throw an error if there's no payload in the request data

Reply:
Nice, one!!! This shows python understanding from your end. Kudos